docs/conf.py

```diff
@@ -235,7 +235,7 @@
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {
     "bids": ("https://bids-standard.github.io/pybids/", None),
```

Comment:
All versions pointed here should probably be synchronized with the versions specified when listing the dependencies, but that is a separate issue/PR.